xbmc/settings/ServicesSettings.cpp

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright (C) 2023 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#include "ServicesSettings.h"
+
+void CServicesSettings::SettingOptionsChunkSizesFiller(const SettingConstPtr& setting,
+                                                       std::vector<IntegerSettingOption>& list,
+                                                       int& current,
+                                                       void* data)
+{
+  list.emplace_back("16 KB", 16);
+  list.emplace_back("32 KB", 32);
+  list.emplace_back("64 KB", 64);
+  list.emplace_back("128 KB", 128);
+  list.emplace_back("256 KB", 256);
+  list.emplace_back("512 KB", 512);
+  list.emplace_back("1 MB", 1024);
```

Comment:
@thexai  Late to the party, sorry. Are you sure that these values should not be translated? is it for countries in the world KB and MB?

Reply:
Do you plan to add translation support (strings.po) in a follow up PR? I think we should do this.

Reply:
I see a translation available for `MByte` but nothing else. It would be nice if we could reuse existing translations if available.

https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L9407

Reply:
> > is it for all countries in the world KB and MB?
> 
> Probably not, although probably in all countries understand what means 128 KB in English

In Romanian, we use octets (octeți) as the translation for bytes so pretty much every software that translates units properly will use that, so it KO, MO, GO and or KiO, MiO, GiO, depending how originally it was written in English.

In Qbittorrent for example, if the language is changed to Romanian, all units of size and time / date are properly translated.

The units in English might be understood by people, but it's most likely that they will be understood by people who already understand english, but the translations are not intended for those people.

And if somebody uses Windows and Qbittorent in Romanian and these display units as properly translated, then it would be nice if Kodi would use the proper units translations too.

Reply:
Done in https://github.com/xbmc/xbmc/pull/24049